storybook/stories/CustomerDetails/index.tsx

```diff
@@ -0,0 +1,91 @@
+import React, { useState, ReactElement } from "react";
+import { storiesOf } from "@storybook/react-native";
+import { InputSelection } from "../../../src/components/CustomerDetails/InputSelection";
+import { InputIdSection } from "../../../src/components/CustomerDetails/InputIdSection";
+import { ManualPassportInput } from "../../../src/components/CustomerDetails/ManualPassportInput";
+import { size, color } from "../../../src/common/styles";
+import { View } from "react-native";
+import { IdentificationFlag } from "../../../src/types";
+
+const selectionArray: IdentificationFlag[] = [
+  {
+    scannerType: "CODE_39",
+    type: "STRING",
+    validation: "NRIC",
+    label: "label_1",
+  },
+  {
+    scannerType: "QR",
+    type: "STRING",
+    validation: "REGEX",
+    label: "label_2",
+  },
+];
+
+function InputSelectionItem(): ReactElement {
```

Comment:
Why use the ES5 function syntax here?

Reply:
in existing storybook we have ES5 syntax. just for consistence followed the same.

Reply:
updated in all places